rinja_parser/src/lib.rs

```diff
@@ -730,7 +730,7 @@ impl Level {
 #[allow(clippy::type_complexity)]
 fn filter(i: &str, level: Level) -> ParseResult<'_, (&str, Option<Vec<WithSpan<'_, Expr<'_>>>>)> {
     let (i, (_, fname, args)) = tuple((
-        char('|'),
+        ws(char('|')),
```

Comment:
The jinja spec doesn't allow whitespace characters around the `|` character, so not sure if we should. However we could definitely add a super nice error in case there are whitespace characters around it.

Reply:
I added a hopefully useful error message for this case.